File: sample/src/main/java/io/snabble/testapp/ShoppingCartActivity.java
package io.snabble.testapp;

import androidx.fragment.app.Fragment;

import io.snabble.sdk.ui.SnabbleUICallback;
import io.snabble.sdk.ui.integration.ShoppingCartFragment;

public class ShoppingCartActivity extends BaseActivity implements SnabbleUICallback {
    @Override
    public Fragment onCreateFragment() {
        return new ShoppingCartFragment();
    }
}

File: ui/src/main/java/io/snabble/sdk/ui/cart/ShoppingCartActivity.kt
package io.snabble.sdk.ui.cart

import androidx.fragment.app.Fragment
import io.snabble.sdk.ui.BaseFragmentActivity

@Deprecated(
    message = "The solo shopping cart view is end-of-life.",
    ReplaceWith(
        "Use the CombinedScannerFragment instead.",
        "io.snabble.sdk.ui.scanner.CombinedScannerFragment"
    )
)
class ShoppingCartActivity : BaseFragmentActivity() {

    override fun onCreateFragment(): Fragment = ShoppingCartFragment()
}
